npm-global-without-sudo.md

```diff
@@ -40,20 +40,12 @@ if [[ ":${MANPATH}:" != *":${NPM_PACKAGES}/share/man:"* ]]; then
 fi
 
 # From https://github.com/npm/npm/issues/6675#issuecomment-168053009
-export NPM_CONFIG_USERCONFIG="$XDG_CONFIG_HOME/.npmrc"
+export NPM_CONFIG_USERCONFIG="${XDG_CONFIG_HOME:-"$HOME"}/.npmrc"
 
 unset NPM_PACKAGES
 ```
 
-If you're using `fish`, add the following to `~/.config/fish/config.fish`:
-
-```sh
-set NPM_PACKAGES "$HOME/.npm-packages"
-
-set PATH $NPM_PACKAGES/bin $PATH
-
-set MANPATH $NPM_PACKAGES/share/man $MANPATH  
-```
+If you're using `fish`, replace in the above code snippet `export` by `set` and add it to `~/.config/fish/config.fish`.
```

Comment:
Are you sure `fish` supports all the syntax used above? For example, `:-`.

// @Slayug

Reply:
No; I found this [resource](https://stackoverflow.com/questions/54524521/in-fish-shell-how-to-set-a-variable-with-default-fallback) and added a commit that expands on it.

Reply:
@sindresorhus no, same case for `if` statement or the way the PATH variable is built.  

I think it's easier to maintain different versions for each shell (less complexity, atomic part) and also for the final users (easier to copy, paste and it works, don't need to be bothered with the other shell).

Reply:
Dear @Slayug I am glad that a fish user chimes in. Could you add the fish version?

Reply:
@Konfekt i have done something, is it possible to push to your forked repository ?

Reply:
@Slayug Thank you very much, it should be possible now.